snippets/csharp/concepts/methods/params75.cs

```diff
@@ -0,0 +1,46 @@
+//<Snippet75>
+using System;
+using System.Linq;
+
+class Example
+{
+    static void Main()
+    {
+        // Invoking the method with an array
+        string fromArray = GetVowels(
+            new[] { "apple", "banana", "pear" });
+
+        // Invoking with a comma-separated list of arguments
+        string fromMultipleArguments = GetVowels(
+            "apple", "banana", "pear");
+
+        // Invoking without any arguments
+        string fromNoValue = GetVowels();
+
+        Console.WriteLine("Vowels in words:");
+        Console.WriteLine($"    From array: '{fromArray}'");
```

Comment:
Nit: putting every `WriteLine` call directly after the corresponding `GetVowels` call would eliminate the need in the comments.

Reply:
Good suggestion! I'll fix that later today